pkg/entities/helm_release_secret.go

```diff
@@ -0,0 +1,29 @@
+package entities
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+type HelmReleaseSecret struct {
+	Secret *v1.Secret
```

Comment:
not sure what is the difference between an example and my code

Reply:
the difference is in embedding. Sometimes it's ok to embed a different structure instead of wrapping if you want to define additional functions for it.

Reply:
done